tests/python/relay/test_to_gnf.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import tvm
+from tvm import relay
+from tvm.relay.ir_pass import to_graph_normal_form, alpha_equal
+from tvm.relay import op, create_executor
+from tvm.relay.backend.interpreter import Value, TupleValue
+
+
+def check_eval(expr, args, expected_result, mod=None, rtol=1e-07):
```

Comment:
write test cases that does ANF -> GNF roundtrip

Reply:
Get an existing neural network, run ToANF, and then ToGraphNormalForm and check consistency

Reply:
is it necessary to do an nn? i get a simpler test working